tests/unit/test_units.py

```diff
@@ -187,3 +187,138 @@ def test_metric_conversion(metric_name, species, conversion):
+
+
+@pytest.mark.parametrize(
+    "metric_name,mixture,conversion",
+    (
+        # mixtures for which the constituents are not reported in SAR/AR4/AR5
+        # wikipedia has values nevertheless, so might come in future ARs
+        ["AR4GWP100", "HCFC402a", 2_788],
+        ["AR4GWP100", "HCFC402b", 2_416],
+        ["AR4GWP100", "HCFC403a", 3_124],
+        ["AR4GWP100", "HCFC403b", 4_457],
+        ["AR4GWP100", "HCFC405a", 5_328],
+        ["AR4GWP100", "HCFC406a", 1_943],
+        ["AR4GWP100", "HCFO411a", 1_597],
+        ["AR4GWP100", "HCFO411b", 1_705],
+        ["AR4GWP100", "HFC413a", 2_053],
+        ["AR4GWP100", "HCFC414a", 1_478],
+        ["AR4GWP100", "HCFC414b", 1_362],
+        ["AR4GWP100", "HCFC416a", 1_084],
+        ["AR4GWP100", "HFC417a", 2_346],
+        ["AR4GWP100", "HFC417b", 3_027],
+        ["AR4GWP100", "HCFC418a", 1_741],
+        ["AR4GWP100", "HFC419a", 2_967],
+        ["AR4GWP100", "HFC422b", 2_526],
+        ["AR4GWP100", "HFC422c", 3_085],
+        ["AR4GWP100", "HFC422d", 2_729],
+        ["AR4GWP100", "HFC424a", 2_440],
+        ["AR4GWP100", "HFC426a", 1_508],
+        ["AR4GWP100", "HFC428a", 3_607],
+        ["AR4GWP100", "HFC429a", 13.9],
+        ["AR4GWP100", "HFC430a", 95],
+        ["AR4GWP100", "HFC431a", 38.3],
+        ["AR4GWP100", "HO432a", 1.64],
+        ["AR4GWP100", "HO433a", 2.85],
+        ["AR4GWP100", "HO433b", 3.23],
+        ["AR4GWP100", "HO433c", 2.93],
+        ["AR4GWP100", "HFC434a", 3_245],
+        ["AR4GWP100", "HFC435a", 25.6],
+        ["AR4GWP100", "HC436a", 3.17],
+        ["AR4GWP100", "HC436b", 3.16],
+        ["AR4GWP100", "HFC437a", 1_805],
+        ["AR4GWP100", "HFC438a", 2_265],
+        ["AR4GWP100", "HFC439a", 1_983],
+        ["AR4GWP100", "HFC440a", 144],
+        ["AR4GWP100", "HC441a", 3.6],
+        ["AR4GWP100", "HFO448a", 1_273],
+        ["AR4GWP100", "HFO449a", 1_282],
+        ["AR4GWP100", "HFO452b", 676],
+        ["AR4GWP100", "HFO454a", 239],
+        ["AR4GWP100", "HFO454b", 466],
+        ["AR4GWP100", "HFO454c", 148],
+        ["AR4GWP100", "HFO455a", 146],
+        ["AR4GWP100", "HFO456a", 687],
+        ["AR4GWP100", "HFO457a", 139],
+        ["AR4GWP100", "HFO459a", 460],
+        ["AR4GWP100", "HCFC500", 8_077],
+        ["AR4GWP100", "HCFC506", 4_495],
+        ["AR4GWP100", "HC510a", 1.24],
+        ["AR4GWP100", "HC511a", 3.19],
+        ["AR4GWP100", "HFO513a", 573],
+    ),
+)
+def test_mixtures_constituents_no_gwp(metric_name, mixture, conversion):
+    with pytest.raises(DimensionalityError):
+        gwp = (  # noqa: F841
+            (1 * unit_registry(mixture)).to("CO2", metric_name).magnitude
+        )
+
+
+def test_mixture_constituent_sum_one():
+    for mixture in unit_registry._mixtures:
+        constituents = unit_registry.split_gas_mixture(1 * unit_registry(mixture))
+        np.testing.assert_allclose(sum((c.magnitude for c in constituents)), 1)
+
+
+def test_split_invalid():
+    with pytest.raises(ValueError):
```

Comment:
are you able to check the messages with e.g. `pytest.raises(ValueError, match="error message that is expected")` please?

Reply:
example here https://github.com/openscm/scmdata/blob/438b78732a971d7eae6631b17e19da13fb6ba403/tests/unit/test_run.py#L108

Reply:
done, thanks for the suggestion!